lib/rules/no-invalid-html-attribute.js

```diff
@@ -254,9 +267,7 @@ function checkLiteralValueNode(context, attributeName, node, parentNode, parentN
     report(context, messages.onlyStrings, 'onlyStrings', {
       node,
       data: { attributeName },
-      fix(fixer) {
-        return fixer.remove(parentNode);
-      },
+      suggestion(fixMessages.removeNonString, fixer, fixModes.remove, parentNode)
```

Comment:
all these lines are invalid syntax - what does `suggestion` return? it seems like maybe the definition of that function is missing, but it'd need to return an object key's *value*, or, an object that'd need to be spread in here.

Reply:
yeah, tried something on these lines